.github/workflows/scorecard.yml

```diff
@@ -20,6 +20,7 @@ permissions: read-all
 jobs:
   analysis:
     name: Scorecard analysis
+    if: "github.repository == 'redhat-cop/org'"
```

Comment:
when i've changed this action yaml in the past, scorecard has detected changes and rejected it, so happy to merge but might well back out the change if it complains on `main`

Reply:
Just for the record, it was anyway more the other one (update org) I was caring about, as it bugs me each night.